index.js

```diff
@@ -1,19 +1,4 @@
 'use strict';
-
-require('dotenv').config({silent: true});
-
-let express = require('express');
-let path = require('path');
-
-// Slack
+require('dotenv').config({ silent: true });
```

Comment:
Good point :ok_hand:. Probably just add it as a dependency in that case. I'll move it to dependencies for the time being.